### ISSUE TITLE
Launcher is not handling corrupted files gracefully, resulting in unrecoverable state

### ISSUE THREAD

**Issue by deathbeam** [runelite/launcher#53]
```
2019-02-15 18:40:10 [main] ERROR net.runelite.launcher.Launcher - unable to download artifacts
java.io.IOException: Data error (cyclic redundancy check)
at java.io.FileInputStream.readBytes(Native Method)
at java.io.FileInputStream.read(FileInputStream.java:233)
at com.google.common.io.ByteStreams.copy(ByteStreams.java:106)
at com.google.common.io.ByteSource.copyTo(ByteSource.java:246)
at com.google.common.io.ByteSource.hash(ByteSource.java:326)
at net.runelite.launcher.Launcher.hash(Launcher.java:367)
at net.runelite.launcher.Launcher.download(Launcher.java:305)
at net.runelite.launcher.Launcher.main(Launcher.java:186)
```

**Comment by gamergui** [comment]
I uninstalled RuneLite, deleted all the folders (in appdata and the .runelite folder). I then re-installed RuneLite onto a new folder on my desktop. Works fine now

**Comment by Adam-** [comment]
fixed in b2464dabc4543c96e7388aea08b4b90a1cc3db7e
